bundle/bundle_test.go

```diff
@@ -62,22 +65,22 @@ func TestBundleCacheDirOverride(t *testing.T) {
 	// now we expect to use 'bundleTmpDir' instead of CWD/.databricks/bundle
 	t.Setenv("DATABRICKS_BUNDLE_TMP", bundleTmpDir)
 
-	cacheDir, err := bundle.CacheDir()
+	cacheDir, err := bundle.CacheDir(ctx)
 
 	// format is <DATABRICKS_BUNDLE_TMP>/<target>
 	assert.NoError(t, err)
 	assert.Equal(t, filepath.Join(bundleTmpDir, "default"), cacheDir)
 }
 
 func TestBundleMustLoadSuccess(t *testing.T) {
-	t.Setenv(envBundleRoot, "./tests/basic")
+	t.Setenv(env.RootVariable, "./tests/basic")
```

Comment:
shall we use envlib.Set(ctx, env.RootVariable, "./tests/basic") here instead already? Or in general what is our plan to get rid of t.Setenv?

Reply:
We have (and will have) a couple of long running tests that should be parallelized. We don't have to parallelise all tests IMO, for example tests that don't make any/very few network calls.

For example, the sync integration tests can take minutes if run serially and thus are important to parallelise.

Reply:
Indeed, we can do this incrementally. I'd like this change to be as small and mechanical as possible.